hypothesis-python/tests/nocover/test_floating.py

```diff
@@ -114,11 +112,6 @@ def test_is_in_exact_int_range(x):
     REALLY_SMALL_FLOAT = sys.float_info.min * 2
```

Comment:
Maybe this is obvious to somebody—me—who didn't just learn about the concept of sub-normals today, but I couldn't figure this out. My guess is that you could use an interpreter built in one architecture but run this in the other architecture, so `sys.float_info` could be incorrect?

Reply:
Nah, it'll be if the interpreter was built with various unsafe compiler flags which flush denormals to zero ([e.g. `-funsafe-math-optimizations`](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html)).  Which is *terrible*, but here we are.

(I'll clarify the test and comment, and make this a skipif)